src/lib/components/AnnouncementBanner/AnnouncementBanner.scss

```diff
@@ -0,0 +1,35 @@
+.usa-banner.ldaf-banner--announcement {
+  background: $primary-darkest;
+
+  .usa-banner__header-text,
+  .usa-banner__guidance {
+    color: white;
+  }
+
+  .usa-banner__button-text {
+    color: $primary-lighter;
+  }
+
+  .ldaf-logo {
+    filter: invert(1);
+  }
+
+  .usa-link {
+    padding-left: 0.25rem;
+    color: $primary-lighter;
+    display: flex;
+    align-items: center;
+  }
+
+  .ldaf-banner--announcement-text {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
Not sure if this is the appropriate block, but it looks like we're not properly centering the announcement as shown in the Figma mock; both the DotGovBanner and AnnouncementBanner look like they're left-aligned within the main container.

Reply:
Courtney made a few adjustments to the design which are reflected here now, as well. Now, it's as follows:
- `DotGovBanner` is always left-aligned
- `AnnouncementBanner` is left-aligned on mobile, centered on on anything wider

Reply:
Seems like the `AnnouncementBanner` is still centered on mobile (at least on the `/news` page). I think it looks fine but based on your comment you might still need to make an update to get it left-aligned on mobile.

Reply:
good catch, should be fixed now